README.md

```diff
@@ -1,5 +1,5 @@
 # DropletKit
-[![Build Status](https://travis-ci.org/digitalocean/droplet_kit.svg?branch=master)](https://travis-ci.org/digitalocean/droplet_kit)
+[![Build Status](https://travis-ci.org/digitalocean/droplet_kit.svg?branch=main)](https://travis-ci.org/digitalocean/droplet_kit)
```

Comment:
Since we use GitHub Actions instead of Travis now, this could be changed to,

```suggestion
[![Build Status](https://github.com/digitalocean/droplet_kit/workflows/CI/badge.svg?branch=main)](https://github.com/digitalocean/droplet_kit/actions)
```

which will render [![Build Status](https://github.com/digitalocean/droplet_kit/workflows/CI/badge.svg?branch=main)](https://github.com/digitalocean/droplet_kit/actions) for now. It should render [![Build Status](https://github.com/digitalocean/droplet_kit/workflows/CI/badge.svg)](https://github.com/digitalocean/droplet_kit/actions) once your PR is merged.